docs/source/install.rst

```diff
@@ -205,3 +121,58 @@ indicates if the commit contained any formatting errors (that are automatically
 	To maintain consitency, we use Black version `22.8.0` (as defined into `.pre-commit-config.yaml`). Make sure to stick to version.  
 
 
+
+
+
+Virtual environment
+==================================
+
+While not mandatory, utilizing a virtual environment when installing disdroDB is Recommended. Using a virtual environment for installing packages provides isolation of dependencies, easier package management, easier maintenance, improved security, and improved development workflow.
+
+
+
+To set up a virtual environment, follow these steps :
+
+* **With venv :**  
+
+	* Windows: Create a virtual environment with venv:
+
+		.. code-block:: bash
+
+		   python -m venv venv
```

Comment:
I would call the virtual environment also here disdrodb-dev !

Reply:
ok done